### PR TITLE
Gather remote changes

### PR DESCRIPTION
**Pull request by almet**
Based on top of #12. The best way to review this is currently to have a look [at the changeset](https://github.com/mozilla-services/kinto-updater/commit/46aa2781c87caa1dc399f2e7328a9009474124e1).

- [x] Code is ready
- [ ] Documentation
- [ ] Tests

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[run]
+omit = kinto_signer/tests/*
+       kinto_signer/hook.py
+       kinto_signer/gather_remote.py
```
Comment (Natim) [inline .coveragerc]: I couldn't find this file, should we remove it from there?

Comment (Natim) [inline .coveragerc]: whereisit

**Comment by Natim** [comment]
r+wc

File: .coveragerc (hunk 1 above)
Comment (almet) [inline .coveragerc]: Hmm, it's actually the file this pull request is adding.

**Comment by almet** [comment]
closing this as it's not useful anymore in our context.

**Pull request closed without merging**